docs/api/odbc/windows.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: docu
+title: ODBC API - Windows
+selected: Windows
+---
+
+# Windows ODBC Driver Manager: Odbccp32.dll
+
+The Microsoft Windows requires an ODBC Driver Manager to manage communication between applications and the ODBC drivers.
+The DM on Windows is provided in a DLL file ***Odbccp32.dll***, and other files and tools.
+For detailed information checkout out the [Common ODBC Component Files](https://docs.microsoft.com/en-us/previous-versions/windows/desktop/odbc/dn170563(v=vs.85)).
+
+
+# Step 1: Download ODBC Driver
+
+DuckDB releases the ODBC driver as asset. For Windows, download it from <a href="https://github.com/duckdb/duckdb/releases/download/v{{ site.currentduckdbversion }}/duckdb_odbc-windows-amd64.zip">Windows Asset</a> that contains the following artifacts:
+
+**duckdb_odbc.dll**: the DuckDB driver compiled for Windows.
+
+**duckdb_odbc_setup.dll**: a setup DLL used by the Windows ODBC Data Source Administrator tool.
+
+**odbc_install.exe**: a installation script to aid the configuration on Windows.
+
+# Step 2: Extracting ODBC artifacts
+
+Unsing PowerShell, run unzip to extract the files:
```

Comment:
Typo in "Unsing" + `unzip` actually is not a powershell command. Maybe just instruct user to unzip .zip file without telling how... 
I'm not sure if my experience in installing the driver is correct and intended: odbc_install.exe didn't copy the dll files to Windows\system32 directory as is my experience e.g. with SQLite driver. It simply pointed registry entries to the location where dlls were extracted. If this is the intended setup, the docs should explicitly instruct the user that the location of extracted files is meant to be permanent and files should not be moved in order the driver keeps working.

Reply:
> Typo in "Unsing" + `unzip` actually is not a powershell command. Maybe just instruct user to unzip .zip file without telling how... I'm not sure if my experience in installing the driver is correct and intended: odbc_install.exe didn't copy the dll files to Windows\system32 directory as is my experience e.g. with SQLite driver. It simply pointed registry entries to the location where dlls were extracted. If this is the intended setup, the docs should explicitly instruct the user that the location of extracted files is meant to be permanent and files should not be moved in order the driver keeps working. Thanks.

@rvasil thanks for your comments.
You are right about the location of the extracted files being permanent.
The first version of the `odbc_install.exe` only focuses on setting up the ODBC registry entries. A new version should copy the .dll files to `Windows\system32` directory.
I will clarify that in the documentation.